remoteworkitem/remoteworkitem_test.go

```diff
@@ -80,7 +81,7 @@ func TestGitHubIssueMapping(t *testing.T) {
 	}
 
 	workItemMap := WorkItemKeyMaps[ProviderGithub]
-	remoteWorkItem := RemoteWorkItem{ID: "xyz", Content: []byte(content)}
+	remoteWorkItem := TrackerItem{Item: string(content[:]), RemoteItemID: "xyz", BatchID: "bxyz", TrackerQueryID: uint64(0)}
```

Comment:
In my opinion, when using a different data structure, we should also rename the corresponding variables. Just a matter of keeping the code tidy.